taskfiles/utils.yml

```
@@ -48,16 +50,19 @@ tasks:
     cmds:
       - task: "compute-checksum"
         vars:
-          DATA_DIR: "{{.DATA_DIR}}"
+          DATA_PATHS:
+            ref: ".DATA_PATHS"
           EXCLUDE_PATHS:
-            ref: "default (list) .EXCLUDE_PATHS"
+            ref: ".EXCLUDE_PATHS"
```

I believe this default fallback is needed for Task `3.40.0`. Was the removal intentional?
```suggestion
            ref: "default (list) .EXCLUDE_PATHS"
```

---

No sorry. I think I might need to add this in other places too.